src/database/mongodb_client.py

```
@@ -192,4 +201,40 @@ def close_connection(self):
+        """Set active keywords for data collection"""
+        try:
+            update_doc = {
+                'active': keywords,
+                'updated_at': datetime.now()
+            }
+            self.db.settings.update_one({'_id': 'keywords'}, {'$set': update_doc})
+            logging.info("Active keywords updated successfully")
+        except Exception as e:
+            logging.error(f"Error setting active keywords: {e}")
+
+    def append_keywords_history(self, old, new, user) -> None:
```

Ruff (0.14.5)

Unused method argument: `new`

(ARG002)